battle-engine.js

```diff
@@ -2045,6 +1955,8 @@ Battle = (() => {
 		this.messageLog = [];
 
 		this.startingSeed = this.generateSeed();
+
+		this.send = send;
```

Comment:
`PM` is an instance of `ProcessManager`. In `simulator.js` it is called `SimulatorProcess`. It is not clear to me how `battle-engine.js` could access `SimulatorProcess.send` without passing around a callback or making `battle-engine.js` dependent on `process-manager.js`.